lib/formatter/csv/classification.rb

```diff
@@ -51,6 +51,10 @@ def subject_data
         end.to_json
       end
 
+      def subject_ids
+          classification.subject_ids.map {|id| cache.subject(id).id }.to_json
```

Comment:
Because it's the cache object's list of ids (relevant to that classification) that we really want. I'm still wrapping my head around how the cache is used, though. Could the current classification's list of subjects be out of date?

And delimiting the list would be just fine with me, if it's easier for their tools to parse.